package.json

```diff
@@ -36,5 +36,8 @@
     "tsconfig-paths": "3.10.1",
     "typescript": "4.3.5",
     "wsrun": "5.2.4"
+  },
+  "dependencies": {
+    "class-validator": "^0.13.2"
```

Comment:
I don't think this belongs in the root package

Reply:
Aren't we using it for both front and back end validation? Should it be in common instead?

Reply:
I think each package that needs it should have it installed, packages in root should only be used for monorepo-specific things (lint/prettier/husky) IMO

Reply:
It might only be required in common